Intersect.Client/Core/Fade.cs

```diff
@@ -64,6 +76,11 @@ public static void Update()
                 {
                     sCurrentAction = FadeType.None;
                     sFadeAmt = 0f;
+
```

Comment:
This comment is for the sFadeAmt -= line: remove the 255 (the way it's written makes it look like it's `x / (duration * 255)` but it's actually `255 * x / duration` -- move the `* 255` into `GetFade()`, make it a property and rename it to `Alpha`

Reply:
So I can see how my previous instruction was confusing -- move the `* 255f` into `GetFade()` and change it into a property called `Alpha`